src/components/text-editor/partial-styles/_prosemirror-menu.scss

```diff
@@ -2,6 +2,21 @@
 
 $size-of-caret: 0.25rem;
 
+div#editor {
+    .ProseMirror-menubar {
+        position: sticky !important;
+        z-index: 1;
+        top: 0;
+
+        &[style*='position: fixed'] {
+            // They add this dynamically as soon as you scroll.
+            // I want to catch this to detect the scroll,
+            // to add proper styles. This is a hack.
+            box-shadow: 0 0.25rem 0.5rem -0.5rem rgb(var(--color-black), 0.8);
```

Comment:
> This is a hack.

You don't say? 😝 🤣

Reply:
"When you know the rules, you are allowed to break the rules" 😉

Reply:
Yeah, I understood. I just couldn't resist poking you because it felt so _utterly_ hackish 😂 🤗